### PR TITLE
feat: add `deep linking` example

### PR DESCRIPTION
**Pull request by marstamm**
showcase the new multi-root capability and drilldown feature of bpmn-js

Features:

- Create deep-linkable locations when using drilldown
- Navigate with the Browser back/forward buttons to change planes

### CODE REVIEW

File: deep-linking/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# bpmn-js collapses subprocesses example
```
Comment (nikku) [inline deep-linking/README.md:1]: Rename to deep-linking example?

File: deep-linking/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "name": "bpmn-js-example-commonjs",
+  "version": "0.0.0",
+  "description": "An example how to bundle an app using bpmn-js for the browser",
+  "main": "src/app.js",
+  "scripts": {
+    "all": "run-s bundle open",
+    "bundle": "webpack",
+    "bundle:watch": "webpack -w",
+    "open": "opn ./public/index.html",
+    "dev": "npm run bundle:watch"
```
Comment (nikku) [inline deep-linking/package.json:11]: My assumption was that `npm start` spawns a demo instance (or `npm run dev` would).

But apparently this is not a pattern for our examples?

File: deep-linking/README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+This example shows how to use the [rootElements API](https://github.com/bpmn-io/diagram-js/blob/master/lib/core/Canvas.js) of [bpmn-js](https://github.com/bpmn-io/bpmn-js) to switch between different diagram layers.
+
+![Screenshot](resources/screenshot.png)
```
Comment (nikku) [inline deep-linking/README.md:5]: Is the padding-left (40px) by design for the breadcrumbns? I did stuble upon it in this example:

image

**Review by nikku — COMMENTED** [review]
Very nice. 

Found some small things to maybe improve upon.

File: deep-linking/README.md (hunk 3 above)
Comment (nikku) [reply deep-linking/README.md:5]: Expected behavior: In the viewer the breadcrumbs start top-left (no padding).

File: deep-linking/package.json (hunk 2 above)
Comment (marstamm) [reply deep-linking/package.json:11]: It does seem like there is no pattern, I just copied and adjusted another example. I'm fine with introducing it though

File: deep-linking/README.md (hunk 3 above)
Comment (nikku) [reply deep-linking/README.md:5]: Let us make it "left padding == top padding or something, visually speaking.

image

Comment (marstamm) [reply deep-linking/README.md:5]: cf. https://github.com/bpmn-io/bpmn-js/pull/1608

I decided to give position it in a way that the modeler only moves the breadcrumbs right but not down. It should work for most viewer integrations, where a Diagram is displayed in some sort of border.

For this PR, we can make it flush with the heading with custom CSS instead.

**Review by nikku — APPROVED** [review]

**Pull request merged**